@xen-orchestra/lite/src/components/PanelHeader.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { withState } from 'reaclette'
+
+import Icon, { IconName } from './Icon'
+
+import Button, {ButtonProps} from '@mui/material/Button';
+import ButtonGroup, {ButtonGroupClassKey}  from '@mui/material/ButtonGroup';
+import Stack from '@mui/material/Stack';
+import Typography , {TypographyClassKey} from '@mui/material/Typography';
+
+interface ParentState {}
+
+interface State {}
+
+interface Action extends ButtonProps{
+  icon: IconName
+}
+
+interface ParentEffects {}
+
+interface Effects {}
+
+interface Computed {}
+
+const DEFAULT_H1_STYLE = {flex: 1}
+const DEFAULT_BUTTONGROUP_STYLE = {margin: '0.5em', flex: 0}
+
+interface Props extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  // Accepts an array of Actions. An action accepts all the props of a Button + an icon
+  actions?: Array<Action>,
+  // the props passed to the title, accepts all the keys of Typography
+  titleProps?: TypographyClassKey,
+  // the props passed to the button group, accepts all the keys of a ButtonGroup
+  buttonGroupProps?: ButtonGroupClassKey
+}
+
+const PanelHeader = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
+  {},
+  ({ actions=[], titleProps = {}, buttonGroupProps={}, children=null }) => (
+    <Stack direction='row' alignItems='start'>
+      <Typography variant="h1" sx={DEFAULT_H1_STYLE} {...titleProps}>
+        {children}
```

Comment:
Should it be a title props instead of children?

Reply:
by using children , I think it allows a richer usage , for example 
```
this is my awesome VM <small> really look at it </small>
```
or adding an icon, a link